test/metabase/api/search_test.clj

```diff
@@ -3,7 +3,7 @@
    [clojure.set :as set]
    [clojure.string :as str]
    [clojure.test :refer :all]
-   [java-time :as t]
+   [java-time.api :as t]
```

Comment:
Did this sneak in somehow?

Reply:
I think this is required since the java-time kondo alias got renamed.

Reply:
I've gone ahead and make the changes elsewhere